### PR TITLE
Quality: Add alias to Notes section of Clear-Host and Out-Host Cmdlets

### LINKED ISSUE #9629: Quality: Add alias to Notes section of Clear-Host and Out-Host Cmdlets
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/3?filterQuery=repo%3A%22MicrosoftDocs%2FPowerShell-Docs%22+is%3Aissue

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Aliases

### Article List

- Clear-Host
- Out-Host

### PR DESCRIPTION
**Pull request by szabolevo**
# PR Summary

This adds alias information to the Notes section in the documentation for Clear-Host and Out-Host.

- Resolves #9629 

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by ehmiiz** [comment]
Awesome PR! 🥳
Some feedback: be sure to edit the Ms.keydate value to the current date, (line 5 in the .med file), to inform others when changes was made to the docs. 
Happy new year's 🎈

File: reference/5.1/Microsoft.PowerShell.Core/Clear-Host.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ This cmdlet returns no output.
 
 ## NOTES
 
+Windows PowerShell includes the following aliases for `Clear-Host`:
+
+- `cls`
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Core/Clear-Host.md:92]:
```suggestion
- `clear`
- `cls`
```

File: reference/7.2/Microsoft.PowerShell.Core/Clear-Host.md [hunk 2]
```diff
@@ -86,6 +86,11 @@ You can't pipe objects to this cmdlet.
 This cmdlet returns no output.
 
 ## NOTES
+ 
+PowerShell includes the following aliases for `Clear-Host`:
+
+- All platforms:
+  - `cls`
```
Comment (sdwheeler) [inline reference/7.2/Microsoft.PowerShell.Core/Clear-Host.md:93]:
```suggestion
- Windows:
  - `clear`
  - `cls`

- Linux and macOS:
  - `cls`
```

File: reference/7.3/Microsoft.PowerShell.Core/Clear-Host.md [hunk 3]
```diff
@@ -87,6 +87,11 @@ This cmdlet returns no output.
 
 ## NOTES
 
+PowerShell includes the following aliases for `Clear-Host`:
+
+- All platforms:
+  - `cls`
```
Comment (sdwheeler) [inline reference/7.3/Microsoft.PowerShell.Core/Clear-Host.md:93]:
```suggestion
- Windows:
  - `clear`
  - `cls`

- Linux and macOS:
  - `cls`
```

File: reference/7.4/Microsoft.PowerShell.Core/Clear-Host.md [hunk 4]
```diff
@@ -87,6 +87,11 @@ This cmdlet returns no output.
 
 ## NOTES
 
+PowerShell includes the following aliases for `Clear-Host`:
+
+- All platforms:
+  - `cls`
```
Comment (sdwheeler) [inline reference/7.4/Microsoft.PowerShell.Core/Clear-Host.md:93]:
```suggestion
- Windows:
  - `clear`
  - `cls`

- Linux and macOS:
  - `cls`
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
@szabolevo Thanks for the PR.

Please accept the suggested changes and update the `ms.date` value at the top of each file.

**Comment by szabolevo** [comment]
Thanks @ehmiiz  and @sdwheeler  for the feedback, fixed both recommendations. Happy New Year!

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**